src/languages/coldfusion.coffee

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+
+  name: "CFML"
```

Comment:
What about changing this to `ColdFusion`? I'd like the `name`s for the languages to be full, and it also helps with documentation that is automatically generated.

Of course, if you change this name, as you know, remember to change in the `beautifiers/prettydiff.coffee`